6.4.remember_remember.py

```diff
@@ -0,0 +1,26 @@
+from PIL import Image
+
+
+def decoding_an_image(path):
+    # Open the image
+    try:
+        img = Image.open(path)
+    except:
```

Comment:
Always throw the most specific exception - 
https://stackoverflow.com/questions/2052390/manually-raising-throwing-an-exception-in-python
https://docs.python.org/3/library/exceptions.html
 In this case you may not even want to use try except

Reply:
fixed it